package/ports_daemon/overlay/flags/can_ports

```diff
@@ -0,0 +1 @@
+0
```

Comment:
This needs to be `overlay/etc/flags/can_ports` to create a file in `/etc/flags`.